SonarScanner.Shim/SonarScanner.Wrapper.cs

```diff
@@ -117,7 +117,16 @@ private static void InternalExecute(AnalysisConfig config, IEnumerable<string> u
         private static string FindScannerExe()
         {
             var binFolder = Path.GetDirectoryName(typeof(SonarScannerWrapper).Assembly.Location);
-            return Path.Combine(binFolder, "sonar-scanner-" + SonarScannerVersion + @"\bin\sonar-scanner.bat");
+            var scannerScript = IsWindowsOperatingSystem()
+                ? @"sonar-scanner.bat"
+                : @"sonar-scanner.sh";
+            return Path.Combine(binFolder, $"sonar-scanner-{SonarScannerVersion}", "bin", scannerScript);
+        }
+
+        private static bool IsWindowsOperatingSystem()
```

Comment:
I went for a simple solution but this [SO thread](https://stackoverflow.com/questions/38790802/determine-operating-system-in-net-core) seems to indicate it doesn't fully work for MacOS so we might need to use the second solution mentioned. But let's try the simple one first.

Reply:
Ok, but why not `return Environment.OSVersion.Platform == PlatformID.Windows` ? I don't like in the current implementation that windows is "not osx and not unix", as if there is no other OS but Windows.

Reply:
@janos-ss because of there are multiple values for windows https://msdn.microsoft.com/en-us/library/3a8hyw88(v=vs.110).aspx

Reply:
@Evangelink I see, makes sense

Reply:
Although in practice, we'll only encounter `Win32NT`. I have small doubts if .NET runs on Windows 95 :)